docs/user/sbt.rst

```diff
@@ -198,6 +198,13 @@ Scala Native supports three distinct linking modes:
    Similar to clang's ``-O2`` with addition of link-time optimization over
    the whole application code.
 
+2. **release-size.** (introduced in 0.5.0)
+
+   Optimize for output size while still trying to keep
```

Comment:
"Optimize for reduced output size while..."

Reply:
thanks, updated